src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -202,6 +216,19 @@ private async ValueTask LoadOrReloadProjectsAsync(ImmutableSegmentedList<string>
         }
     }
 
+    private Build.Framework.ILogger? CreateMSBuildLogger()
+    {
+        if (_globalOptionService.GetOption(LanguageServerProjectSystemOptionsStorage.BinaryLogPath) is not string binaryLogDirectory)
```

Comment:
see other PR- this could potentially just be a boolean option and we write the binary log to the extension logs folder.

Reply:
Deciding to keep this as is.